components/kyma-environment-broker/internal/process/provisioning/audit_log_step_test.go

```diff
@@ -0,0 +1,147 @@
+package provisioning
+
+import (
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/auditlog"
+
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/process/provisioning/automock"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage"
+	"github.com/kyma-incubator/compass/components/provisioner/pkg/gqlschema"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAuditLog_ScriptFileDoesNotExist(t *testing.T) {
+	// given
+	mm := afero.NewMemMapFs()
+
+	repo := storage.NewMemoryStorage().Operations()
+	cfg := auditlog.Config{
+		URL:      "host1",
+		User:     "aaaa",
+		Password: "aaaa",
+		Tenant:   "tenant",
+	}
+	svc := NewAuditLogOverridesStep(repo, cfg)
+	svc.fs = mm
+
+	operation := internal.ProvisioningOperation{
+		ProvisioningParameters: `{"ers_context": {"subaccount_id": "1234567890"}}`,
+	}
+	repo.InsertProvisioningOperation(operation)
+
+	// when
+	_, _, err := svc.Run(operation, NewLogDummy())
+	//then
+	require.Error(t, err)
+	require.EqualError(t, err, "open /audit-log-script/script: file does not exist")
+
+}
+
+func TestAuditLog_HappyPath(t *testing.T) {
+	// given
+	mm := afero.NewMemMapFs()
+
+	fileScript := `
+func myScript() {
+foo: sub_account_id
+bar: tenant_id
+return "fooBar"
+}
+`
+
+	err := afero.WriteFile(mm, "/audit-log-script/script", []byte(fileScript), 0755)
+	if err != nil {
+		t.Fatalf("Unable to write contents to file: audit-log-script!!: %v", err)
+	}
+
+	repo := storage.NewMemoryStorage().Operations()
+	cfg := auditlog.Config{
+		URL:      "https://host1:8081/aaa/v2",
```

Comment:
please use a different port to see that it really works, as this port is hardcoded in code ;)

Reply:
fixed.